aws_crt_common_ws/aws-crt-common/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "aws-crt-common"
+version = "0.1.0"
+authors = ["AWS Common Runtime Team (aws-sdk-common-runtime-team@amazon.com)"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[build-dependencies]
+
+[dependencies]
+aws-crt-common-sys = { path = "../aws-crt-common-sys" }
+futures = "0.3"
```

Comment:
It *looks* like you are only using `Waker`. `futures::task::Waker` is a re-export of `std::task::Waker`, so you should not need the `futures` crate here. The `futures` crate is pretty heavy. If you *do* need functionality only provided by `futures`, you probably will want to depend on the specific sub-crates (e.g. `futures-core`).